src/ape_run/_cli.py

```diff
@@ -75,7 +82,13 @@ def _get_command(self, filepath: Path) -> Union[click.Command, click.Group, None
             def call(network):
                 _ = network  # Downstream might use this
                 with use_temp_sys_path(filepath.parent.parent):
-                    ns = run_module(f"scripts.{filepath.stem}")
+                    path = (
+                        str(filepath)
+                        .replace(os.path.sep, ".")
+                        .split("scripts.")[-1]
+                        .replace(filepath.suffix, "")
```

Comment:
I think this could potentially cause issues if the suffix appears in another part of the path like `scripts.pyfiles.script.py`

Reply:
```python
                    path = ".".join(
                        (
                            str(filepath)
                            .replace(os.path.sep, ".")
                            .split("scripts.")[-1]
                            .split(".")[:-1]
                        )
                    )
```

Reply:
made this into helper method